sneakers/channels/googleSpread.py

```diff
@@ -0,0 +1,74 @@
+# Authors: Gabriel Butterick and Bonnie Ishiguro
+
+import time
+import json
+import gspread
+from oauth2client.client import SignedJwtAssertionCredentials
+
+from sneakers.modules import Channel
+
+class GoogleSpread(Channel):
+    description = """\
+        Posts data to Google Spreadsheets.
+    """
+
+    requiredParams = {
+        'sending': {
+            'client_email': '',
+            'private_key': '',
+            'google_sheet': ''
+        },
+        'receiving': {
+            'client_email': '',
+            'private_key': '',
+            'google_sheet': ''
+        }
+    }
+
+    # Courtesy Limit: 10,000,000 queries/day
```

Comment:
It might be a good idea to set a maxLength and maxHourly parameter. Take a look at another channel to see how it's used there.